lib/block_view_stake.go

```diff
@@ -1469,6 +1457,9 @@ func (bav *UtxoView) _connectUnstake(
 		}
 	}
 	// 3. Delete the PrevLockedStakeEntry, if exists.
+	//
+	// Note that we don't technically need to do this since the flush will naturall delete
```

Comment:
```suggestion
	// Note that we don't technically need to do this since the flush will naturally delete
```

Reply:
Addressed in https://github.com/deso-protocol/core/pull/531.